service/test/lib/integration/teamOpsTest.js

```diff
@@ -5,188 +5,259 @@ const _ = require('lodash')
 const expect = require('code').expect
 const Lab = require('lab')
 const lab = exports.lab = Lab.script()
+const crypto = require('crypto')
 
 const teamOps = require('../../../lib/ops/teamOps')
 const policyOps = require('../../../lib/ops/policyOps')
 const userOps = require('../../../lib/ops/userOps')
 
-let testTeamId
-const teamData = {
-  name: 'Team 4',
-  description: 'This is a test team',
-  parentId: null,
-  organizationId: 'WONKA'
-}
-const updateData = {
-  name: 'Team 5',
-  description: 'description',
-  organizationId: 'WONKA'
-}
-const updateTeamMembersData = {
-  users: ['MikeId', 'CharlieId'],
-  organizationId: 'WONKA'
+function randomId () {
+  return crypto.randomBytes(2).toString('hex')
 }
 
-
 lab.experiment('TeamOps', () => {
 
-  lab.before((done) => {
-    teamOps.createTeam(teamData, function (err, result) {
```

Comment:
Can you elaborate on why the removal of the team creation? I was thinking we could use it (and maybe add another one) for the `update`/`read`/`nest`/`unnest`. 

As I see it now, we have a lot of `teamOps.createTeam`s making the tests longer to read and a bit more confusing.